Microsoft.Toolkit.Uwp.Services/Services/Bing/BingDataProvider.cs

```diff
@@ -41,7 +42,7 @@ public class BingDataProvider : DataProviderBase<BingSearchConfig, BingResult>
         protected override async Task<IEnumerable<TSchema>> GetDataAsync<TSchema>(BingSearchConfig config, int maxRecords, IParser<TSchema> parser)
         {
             var countryValue = config.Country.GetStringValue();
-            var locParameter = string.IsNullOrEmpty(countryValue) ? string.Empty : $"loc:{countryValue}+";
+            var locParameter = string.IsNullOrEmpty(countryValue) ? $"loc:{CultureInfo.CurrentCulture.Name.Split('-')[1].ToLower()}+" : $"loc:{countryValue}+";
```

Comment:
I'm not sure right now but as far as I remember I found something very interesting about TwoLetterISOLanguageName before adding the line like that. Not sure what it was though. Will research again and let you know.